rust/macros/module.rs

```diff
@@ -543,18 +544,28 @@ pub(crate) fn module(ts: TokenStream) -> TokenStream {
             }};
 
             // Loadable modules need to export the `{{init,cleanup}}_module` identifiers.
+            /// # Safety
+            /// This is a exported identifier that should only be called by the kernel at load time.
```

Comment:
Initially I was going to comment that this safety section does not really explain much about the safety preconditions, but I guess it is actually a precondition, in the sense that it does not allow anybody else to call it, so it is fine.

In any case, nit on formatting -- the convention we use is:

```suggestion
            ///
            /// # Safety
            ///
            /// This is a exported identifier that should only be called by the kernel at load time.
```

Reply:
Done, fixed every `# Safety` comment